src/routes/paymentRouter.ts

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+import { Router } from 'express';
+import { isAuthenticated } from '../middlewares/authMiddlewares';
+import { handlePayments } from '../controllers/handlePaymentsController';
+
+export const paymentRouter = Router();
+
+paymentRouter.post('/:orderId/charge', isAuthenticated, handlePayments);
```

Comment:
I think this route should be made so that the parameters are always the lasts. In that way, it would not confuse others, because I probaly think it is the standard naming convention.

Reply:
the way i named this router is because the order comes before the payment, that's why i named it like that and i guess that's not a convention it depends on how you want to name your route